src/core/app_messages.tf

```
@@ -32,6 +38,59 @@ locals {
   }
 }
 
+module "redis_messages_snet" {
+  source                                         = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v2.0.26"
+  name                                           = format("%s-redis-messages-snet", local.project)
+  address_prefixes                               = ["10.0.14.0/25"] # To check
+  resource_group_name                            = data.azurerm_resource_group.vnet_common_rg.name
+  virtual_network_name                           = data.azurerm_virtual_network.vnet_common.name
+  enforce_private_link_endpoint_network_policies = true
+}
+
+module "redis_messages" {
+  source                = "git::https://github.com/pagopa/azurerm.git//redis_cache?ref=v2.0.26"
+  name                  = format("%s-redis-messages-std", local.project)
+  resource_group_name   = azurerm_resource_group.app_messages_rg1.name
+  location              = azurerm_resource_group.app_messages_rg1.location
```

```suggestion
  resource_group_name   = azurerm_resource_group.app_messages_common_rg.name
  location              = azurerm_resource_group.app_messages_common_rg1.location
```
we need a new shared resource group (app-messages-common) between app1 and app2